internal/configs/sharding/shardingconfig.go

```diff
@@ -47,3 +59,37 @@ type Instance interface {
 	// ReshardingEpoch returns a list of Epoch while off-chain resharding happens
 	ReshardingEpoch() []*big.Int
 }
+
+// TxThrottleFlag indicates the throttling flag for a particular transaction
+type TxThrottleFlag int
+
+// Enum for different TxThrottleFlag
```

Comment:
any explanation on those enum?

Reply:
added in the type def above for TxThrottleFlag I can move it here